### PR TITLE
[KO] Add Spun Out

### PR DESCRIPTION
**Pull request by Ge-n-z**

### CODE REVIEW

File: wiki/Game_modifier/Spun_Out/ko.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+태그:
```
Comment (TPGPL) [inline wiki/Game_modifier/Spun_Out/ko.md:2]:
```suggestion
stub: true
tags:
```

File: wiki/Game_modifier/Spun_Out/ko.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - 스펀 아웃
+  - spun out
+  - SO
+  - mod
+  - game modifier
+---
+
+# 스펀 아웃 (모드)
+
+![SO mod icon](/wiki/shared/mods/SO.png "스핀 아웃(SO) 모드 아이콘")
```
Comment (TPGPL) [inline wiki/Game_modifier/Spun_Out/ko.md:12]: `SO mod icon` needs to be translated

File: wiki/Game_modifier/Spun_Out/ko.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- 종류: Special
+- 점수 배수: 0.90x
+- 기본 단축키: `C`
+- 설명: `스피너가 자동으로 돌아갑니다.`
+- SO를 사용할 수 있는 osu!의 장르 ![][o!s]
+
+## 상세 설명
+
+**스펀 아웃**은 [osu!standard](/wiki/Game_mode/osu!)의 플레이어가 [비트맵](/wiki/Beatmaps)을 플레이할 때 성공적으로 스피너를 돌릴 수 있게 도와주는 [모드](/wiki/Game_modifier)입니다.
+이 모드는 모든 비트맵의 스피너를 시계 방향으로 분당 회전수(SPM)를 정확히 286.48로 맞추며 돌립니다. 이 수치는 477.26 SPM의 3/5입니다.
```
Comment (TPGPL) [inline wiki/Game_modifier/Spun_Out/ko.md:28]:
```suggestion
**스펀 아웃**은 [osu!standard](/wiki/Game_mode/osu!)의 플레이어가 [비트맵](/wiki/Beatmaps)을 플레이할 때 성공적으로 스피너를 돌릴 수 있게 도와주는 [모드](/wiki/Game_modifier)입니다. 이 모드는 모든 비트맵의 스피너를 시계 방향으로 분당 회전수(SPM)를 정확히 286.48로 맞추며 돌립니다. 이 수치는 477.26 SPM의 3/5입니다.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Game_modifier/Spun_Out/ko.md (hunk 2 above)
Comment (Ge-n-z) [reply wiki/Game_modifier/Spun_Out/ko.md]: Done :)

**Review by TPGPL — APPROVED** [review]

**Pull request merged**